src/evm-block-extractor/src/database/postgres_db_client.rs

```diff
@@ -169,6 +172,30 @@ impl DatabaseClient for PostgresDbClient {
         Ok(())
     }
 
+    async fn insert_certified_block_data(&self, response: CertifiedBlock) -> anyhow::Result<()> {
+        let block_id = response.data.number.0.as_u64();
+
+        let mut tx = self.pool.begin().await?;
+        sqlx::query("INSERT INTO CERTIFIED_EVM_BLOCK (id, certified_response) VALUES ($1, $2) ON CONFLICT (id) DO UPDATE SET certified_response = $2")
+                    .bind(block_id as i64)
+                    .bind(serde_json::to_value(response)?)
+                    .execute(&mut *tx)
+                    .await
+                    .map_err(|e| anyhow::anyhow!("Error inserting certified block {}: {:?}", block_id, e))
+                    .map(|_| ())?;
+        tx.commit().await?;
+
+        Ok(())
+    }
+
+    async fn get_last_certified_block_data(&self) -> anyhow::Result<CertifiedBlock> {
```

Comment:
I don't think a new table is needed. We have to provide only one certified block at a time, so the single information can be saved in the `EVM_KEY_VALUE_DATA` table which is there for this specific purpose